### DISCUSSION TITLE
Will the app be published in the Microsoft Store?

### DISCUSSION THREAD

**Discussion by VIPsp1** [HerMajestyDrMona/Windows11DragAndDropToTaskbarFix#39] [Q&A]
Hi, I really liked this app! ❤️
Should we expect this program in the **Microsoft Store**?

**Comment by HerMajestyDrMona** [comment]
Hi.

No. There's no need for that, especially because the registration process for the Microsoft developer account is paid. I was trying to do it a few days ago, but there were so many problems on Microsoft sites that I gave up.

Maybe I'll try again one day, but I honestly don't see a point ot it. This program won't even longer be needed in 10 months or so...
